scripts/plotting/tape_recorder.py

```diff
@@ -110,6 +85,19 @@ def tape_recorder(adfobj):
     redo_plot = adfobj.get_basic_info('redo_plot')
     print(f"\t NOTE: redo_plot is set to {redo_plot}")
     #-----------------------------------------
+
+    #This may have to change if other variables are desired in this plot type?
+    plot_name = plot_loc / f"Q_ANN_TapeRecorder_Mean.{plot_type}"
+    print(f"\t - Plotting annual tape recorder for Q")
+
+    # Check redo_plot. If set to True: remove old plot, if it already exists:
+    if (not redo_plot) and plot_name.is_file():
+        #Add already-existing plot to website (if enabled):
+        adfobj.add_website_data(plot_name, "tape_recorder", None, season="ANN", multi_case=True)
+        return
```

Comment:
I grabbed this from one of the other plotting scripts (LatLon?) we don't display it elsewhere. Should I go ahead and make a print statement here and then every other plotting scheme (LatLon, Zonal, etc.) as well?

Reply:
I personally think so, although if we are worried about printing too much to the screen then we can always use `adf.debug_log()` instead of just `print()`, as that way it will only print the messages if a user passes in the `--debug` flag when calling `run_adf_diag`.

Reply:
I added the `debug_log()` here and to all other applicable plotting scripts.